youtube_dl/extractor/rutube.py

```diff
@@ -44,6 +48,11 @@ class RutubeIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    @classmethod
+    def suitable(cls, url):
+        parts = compat_urllib_parse_urlparse(url)
+        return super(RutubeIE, cls).suitable(url) and not parts.query
```

Comment:
If you enter the URL, it doesn't display the playlist on the web interface; both `pl_id` and `pl_type=source` must be given. A URL with `pl_id` and without `pl_type=source` may never be given.